doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -45,8 +45,9 @@ The supported backend and peripheral drivers:
 Initialization
 **************
 
-If the application configures the ``zephyr,host-cmd-backend`` chosen node, then the backend
-automatically initializes the host command subsystem by calling :c:func:`ec_host_cmd_init`.
+If the application configures the ``zephyr,host-cmd-espi-backend``/``zephyr,host-cmd-shi-backend``/
+``zephyr,host-cmd-uart-backend`` chosen node, then the backend automatically initializes the host command
+subsystem by calling :c:func:`ec_host_cmd_init`.
 
 If ``zephyr,host-cmd-backend`` is not chosen, the :c:func:`ec_host_cmd_init` function should be
```

Comment:
Suggestion
"If no backend chosen node is configured, the application must call the `ec_host_cmd_init` function directly.  This way of..."